src/systems/userinput/devices/daydream-controller.js

```diff
@@ -45,6 +57,19 @@ export class DaydreamControllerDevice {
       this.pose.direction.set(0, 0, -1).applyQuaternion(this.rayObjectRotation);
       this.pose.fromOriginAndDirection(this.pose.position, this.pose.direction);
       frame[paths.device.daydream.pose] = this.pose;
+
+      this.headObject3D = this.headObject3D || document.querySelector("#player-camera").object3D;
```

Comment:
Since we are already getting the VRDisplay we might as well do the right thing and directly `getPose()` from it and pass the `VRPose` and `GamepadPose` to `applyArmModel`